introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -287,6 +279,60 @@
     "Note that the \"validation\" channel has been initialized too. The SageMaker XGBoost algorithm actually calculates RMSE and writes it to the CloudWatch logs on the data passed to the \"validation\" channel."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plotting evaluation metrics\n",
+    "Evaluation metrics for the completed training job are available in CloudWatch. We can pull the area under curve metric for the validation data set and plot it to see the performance of the model over time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "metric_name = 'validation:auc'\n",
+    "\n",
+    "plot_start = datetime.datetime.now() - datetime.timedelta(minutes=60)\n",
+    "plot_end = datetime.datetime.now()\n",
+    "\n",
+    "metrics_response = client.get_metric_statistics(\n",
```

Comment:
Much easier and more reliable way to do this:

```
from sagemaker.analytics import TrainingJobAnalytics
metrics = TrainingJobAnalytics(training_job_name=job_name, metric_names=[metric_name]).dataframe()
```

Reply:
Much simpler! Updated to use TrainingJobAnalytics